std/typecons.d

```diff
@@ -3059,25 +3061,31 @@ struct Nullable(T)
         _isNull = false;
         return this;
     }
+    /// ditto
+    Nullable opAssign()(typeof(null) value)
+    {
+        if(value !is null)
```

Comment:
Why is this here? Is it supposed to ever fall here?

Reply:
I don't know what `typeof(null)` is on different platforms. I _assume_ it's going to be just for `null`, but there might be some obscure case later on. Though, I do think it's better to replace this entire `if` with `assert(value is null)`. Thank you!

Reply:
No, `typeof(null)` converts to all pointers type and class types but nothing converts explictly to `typeof(null)`. The only value of `typeof(null)` is null itself.